src/components/@shared/atoms/Container.module.css

```diff
@@ -7,13 +7,13 @@
   width: 100%;
 }
 
-@media (min-width: 40rem) {
+@media (min-width: 55rem) {
```

Comment:
what's the goal of this change? Why change this spacing switch for each and every page we have, as this component here is used in many places?

This change e.g. on iPad 9.7" gives less consistent spacing to the left/right of content.

This is v4, which we want, where the space left/right if exactly half of the space between the elements:

<img width="573" alt="Screen Shot 2022-04-06 at 12 52 17" src="https://user-images.githubusercontent.com/90316/161968714-a3ca5765-1e08-43b3-b487-aee9fe992ccf.png">

And this is this PR reducing all left/right spacing making it feel cramped:

<img width="572" alt="Screen Shot 2022-04-06 at 12 53 31" src="https://user-images.githubusercontent.com/90316/161968883-82a2d67d-4c95-4e6c-abe2-9a5b87fe877d.png">

Reply:
yes, you're right, I just removed it